Sources/Transport/Streams/ProgramStream/ProgramStream.swift

```diff
@@ -20,13 +22,27 @@ extension ProgramStream {
 }
 
 public enum SecurityLayer {
-    case none, tls
+    case none
+    case tls(TLSConfig)
+}
+
+extension SecurityLayer: Equatable {
+    public static func ==(lhs: SecurityLayer, rhs: SecurityLayer) -> Bool {
+        switch (lhs, rhs) {
+        case (.none, .none):
+            return true
+        case (.tls(_), .tls(_)):
+            return true
```

Comment:
I though we did, can't find it now. I removed the `Equatable` conformance for now. When is the next TLS release coming? Without it we can't make this PR pass.